dockers/docker-fpm-frr/supervisord.conf

```diff
@@ -39,6 +39,15 @@ startsecs=0
 stdout_logfile=syslog
 stderr_logfile=syslog
 
+[program:staticd]
+command=/usr/lib/frr/staticd -A 127.0.0.1
+priority=4
+autostart=false
+autorestart=false
+startsecs=0
+stdout_logfile=syslog
+stderr_logfile=syslog
```

Comment:
@lguohan @pavel-shirshov can we force this fix with a separate PR?

Reply:
Why?

Reply:
@pavel-shirshov our regression fails because frr is missing staticd which is essential part of this suite

Reply:
@nazarii-gnydyn 
Can you please share some output?
On our side I see: 
```
# ps ax
  PID TTY      STAT   TIME COMMAND
    1 pts/0    Ss+    0:01 /usr/bin/python /usr/bin/supervisord
   25 pts/0    Sl     0:00 /usr/sbin/rsyslogd -n
   30 pts/0    S      0:00 python /usr/bin/bgpcfgd
   33 pts/0    Sl     0:08 /usr/lib/frr/zebra -A 127.0.0.1 -s 90000000 -M fpm
   36 pts/0    S      0:00 /usr/lib/frr/staticd -A 127.0.0.1
   39 pts/0    Sl     0:02 /usr/lib/frr/bgpd -A 127.0.0.1
   47 pts/0    Sl     0:27 fpmsyncd
   49 pts/1    Ss     0:00 bash
   65 pts/1    R+     0:00 ps ax
```

Reply:
@pavel-shirshov On our side (without this fix):
root@sonic:/home/admin# docker exec -ti bgp bash
root@sonic:/# ps -aux
USER       PID %CPU %MEM    VSZ   RSS TTY      STAT START   TIME COMMAND
root         1  0.5  0.2  58600 20444 pts/0    Ss+  13:03   0:00 /usr/bin/python /usr/bin/supervisord
root        29  0.0  0.0 250136  2860 pts/0    Sl   13:03   0:00 /usr/sbin/rsyslogd -n
root        34  0.0  0.1  45556 13788 pts/0    S    13:03   0:00 python /usr/bin/bgpcfgd
frr         37  0.1  0.0 298552  5884 pts/0    Sl   13:03   0:00 /usr/lib/frr/zebra -A 127.0.0.1 -s 90000000 -M fpm
frr         39  0.1  0.1 299900  8964 pts/0    Sl   13:03   0:00 /usr/lib/frr/bgpd -A 127.0.0.1
root        45  0.5  0.0 101356  4096 pts/0    Sl   13:03   0:00 fpmsyncd
root        47  2.5  0.0  18184  3152 pts/1    Ss   13:05   0:00 bash
root        53  0.0  0.0  36632  2748 pts/1    R+   13:05   0:00 ps -aux
root@sonic:/# supervisorctl status
bgpcfgd                          RUNNING   pid 34, uptime 0:02:00
bgpd                             RUNNING   pid 39, uptime 0:01:59
fpmsyncd                         RUNNING   pid 45, uptime 0:01:59
rsyslogd                         RUNNING   pid 29, uptime 0:02:00
start.sh                         EXITED    May 17 01:03 PM
vtysh_b                          STOPPED   Not started
zebra                            RUNNING   pid 37, uptime 0:01:59